### PR TITLE
[Manual backport] Backport 1521 to v1.1 branch

### PR DESCRIPTION
**Pull request by ru-fu**
Manual backport from https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1521

### CODE REVIEW

File: doc/nrf/gs_ins_windows.rst [hunk 1]
```diff
@@ -44,7 +44,7 @@ We recommend installing according to :ref:`zephyr:windows_install_native`.
 Installing the toolchain
 ************************
 
-To be able to cross-compile your applications for Arm targets, you must install  version 7-2018-q2-update of the `GNU Arm Embedded Toolchain`_.
+To be able to cross-compile your applications for Arm targets, you must install  version 8-2019-q3-update of the `GNU Arm Embedded Toolchain`_.
```
Comment (SebastianBoe) [inline doc/nrf/gs_ins_windows.rst]: We want our users to use the same version as we use in CI.

In CI we use "7-2018-q2" for GNU arm Embedded, so this should stay as-is.

Eventually, we should bump the version in CI and also here, but this is not the right time to do so.

Comment (SebastianBoe) [reply doc/nrf/gs_ins_windows.rst]: Actually, I might be wrong, it appears Thomas has bumped the version in CI without me realizing. 

But I would like to see him, or someone else, confirm this.

Comment (ru-fu) [reply doc/nrf/gs_ins_windows.rst]: @shantha-14 told me the exact opposite, that we use 8-2019-q3 in the CI.

Comment (shantha-14) [reply doc/nrf/gs_ins_windows.rst]: please note documentation was not updated when the docker was updated. I have checked our docker files.this is correct.

**Pull request merged**